mbed/mbed_cdc.py

```diff
@@ -0,0 +1,111 @@
+
+def mbed_cdc(port, reset=False, sterm=False, baudrate=9600, timeout= 10, print_term_header=True):
+    try:
+        from serial import Serial, SerialException
+        import serial.tools.miniterm as miniterm
+    except (IOError, ImportError, OSError):
+        return False
+
+    def get_instance(*args, **kwargs):
```

Comment:
This function does not capture any state. Could you move it outside of the body of the `mbed_cdc` function?

Reply:
It does - see `port` in printed message.

Reply:
port is the first parameter.

Reply:
This is just a minor style comment. Not a blocker.